# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/test/mochitest/create_notification.html
# Commit: 53373d35978a
# Full Hash: 53373d35978a5445344404737fc70f8929830102
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2025-03-12 03:55:25
# Regressor Bug: 1948339
# File Overlap Count: 2
# Description:
#   Bug 1948339 - Part 2: Add NotificationObserver r=asuth
#   
#   This also makes page-bound notifications to not be stored via NotificationStorage.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D240642
# ==============================================================================

diff -r 96d954eae2a0 -r 53373d35978a dom/notification/test/mochitest/create_notification.html
--- a/dom/notification/test/mochitest/create_notification.html	Tue Mar 11 22:23:52 2025 +0000
+++ b/dom/notification/test/mochitest/create_notification.html	Tue Mar 11 22:24:26 2025 +0000
@@ -10,7 +10,10 @@
 // https://github.com/BenoitZugmeyer/eslint-plugin-html/issues/139
 (async () => {
   await NotificationTest.allowNotifications();
-  new Notification("This is a title", {
+
+  let swr = await navigator.serviceWorker.register("notification_empty_sw.js");
+  await navigator.serviceWorker.ready;
+  await swr.showNotification("This is a title", {
     body: "This is a notification body",
     tag: "sometag",
   });